rocev2/src/main/scala/rdma/Headers.scala

```diff
@@ -21,7 +21,8 @@ abstract class RdmaHeader() extends Bundle {
 
 // 48 bytes
 case class BTH() extends RdmaHeader {
-  val opcodeFull = Bits(TRANSPORT_WIDTH + OPCODE_WIDTH bits)
```

Comment:
in order to assign values to opcode in sim, I change the opcodeFull to a function and set opcode as val. Maybe the original design consider some relationship between opcode and trans,  but  there are not constants fot opcodeFull assigment, so I do this change for passing my sim.

Reply:
OK